setup.cfg

```diff
@@ -0,0 +1,101 @@
+# This file is used to configure your project.
+# Read more about the various options under:
+# https://setuptools.pypa.io/en/latest/userguide/declarative_config.html
+# https://setuptools.pypa.io/en/latest/references/keywords.html
+
+[metadata]
+name = safe-security-logger
+description = Add a short description here!
```

Comment:
Let's change this placeholder to an actual short description! :)

Reply:
Updated